src/System.Management.Automation/engine/MergedCommandParameterMetadata.cs

```diff
@@ -500,47 +500,27 @@ internal string GetParameterSetName(uint parameterSet)
 
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-windows`
WSMan Config Provider tab complete tests.Tab completion gets dynamic parameters for '<empty>' using '-co'
```powershell
Expected a collection with size 1, but got collection with size 2 @(System.Management.Automation.CompletionResult, System.Management.Automation.CompletionResult).
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\TabCompletion\TabCompletion.Tests.ps1: line 1363
1363:         $res.CompletionMatches | Should -HaveCount $expected.Count
```